src/main/resources/config/liquibase/changelog/20200706223010_changelog.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="balazsczopf" id="20200706223010">
+        <createTable tableName="achievement">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="achievementPK"/>
+            </column>
+            <column name="title" type="VARCHAR(45)"/>
+            <column name="description" type="VARCHAR(255)"/>
```

Comment:
Is 255 chars enough for the whole achievement description?

Reply:
I would guess so, I mean the exemplary description "Get enough points in the EIST course to receive an achievement while also receiving enough points in PGdP to be a real coding master. And you submitted all your solutions for all exercises right after their releases. Now even Stefan approves your efforts." is exactly 255 characters long. 😄